File: week3/Python/1502. Can Make Arithmetic Progression From Sequence.py
class Solution:
    def canMakeArithmeticProgression(self, arr: List[int]) -> bool:
        surya = sorted(arr)
        d= surya[1]-surya[0]
        d1=0
        for i in range(2,len(surya)):
            d1 = surya[i] - surya [i-1]
            if(d1 != d):
                return False
        return True
        1502. Can Make Arithmetic Progression From Sequence 

File: week3/Java/1502. Can Make Arithmetic Progression From Sequence.java
class Solution {
    public boolean canMakeArithmeticProgression(int[] arr) {
        int[] surya = new int[arr.length];
        for(int i = 0; i<arr.length;i++){
            surya[i] = arr[i];
        }
        Arrays.sort(surya);
        int d = surya[1]-surya[0];
        int d1 = 0;
        for (int i=2;i<arr.length;i++){
            d1=surya[i]-surya[i-1];
            if(d!=d1)
                return false;
        }
        return true;
    }
}

